### PR TITLE
Issue #391, added toggle animation

### PR DESCRIPTION
**Pull request by chrisahhh**
Added toggle animation using transform + transition
Breaks backwards compatibility, the toggle handle is now :after rather :before

### CODE REVIEW

File: src/components/Toggle/Toggle.scss [hunk 1]
```diff
@@ -76,36 +79,59 @@
...
   &:before {
+    content: '';
+    width: 0;
+    position: absolute;
+    height: 16px;
+    top: 0;
+    left: 0;
+    background-color: $ms-color-themeDark;
+    outline: 2px solid transparent;
+    transition: width 0.3s;
```
Comment (mikewheaton) [inline src/components/Toggle/Toggle.scss]: Could we increase the speed to 0.167s and use one of the easing curves from our [current animation classes](http://dev.office.com/fabric/styles#animations)?

**Comment by mikewheaton** [comment]
@philworthington Could we use the Toggle animation from the MDL2 branch on our current Toggle control?

**Comment by philworthington** [comment]
Yeah we most likely need guidance around things like animation for the toggle. The mdl2 branch includes an animation inspired by the one used for checkboxes also. There's been no input from design as to what these animations should actually look like however.

**Comment by mikewheaton** [comment]
@chrisahhh Could you merge the latest master into your branch to resolve the merge conflicts? If we can test it and the animation works I think we'll go ahead and merge, although we will likely need to tweak things later when we have more design input here.

**Comment by mikewheaton** [comment]
Unfortunately we can't merge this PR. Given that the Toggle animation has already been done in the updated design language (to be released in Fabric 4.0) I'm going to close this. Thank you for the contribution – if you'd like to open a new PR based on the current master we can most likely merge it.

**Pull request closed without merging**